Hippo.Core/Domain/QueuedEvent.cs

```diff
@@ -38,10 +38,12 @@ public static class Actions
         public const string CreateAccount = "CreateAccount";
         public const string UpdateSshKey = "UpdateSshKey";
         public const string AddAccountToGroup = "AddAccountToGroup";
+        public const string CreateGroup = "CreateGroup";
 
         public const string RegexPattern = CreateAccount
             + "|" + UpdateSshKey
-            + "|" + AddAccountToGroup;
+            + "|" + AddAccountToGroup
+            + "|" + CreateGroup;
```

Comment:
Getting close to the point where a string.join() with a pipe delimiter would be cleaner.

Reply:
Can't do that. It needs to be a `const` expression in order to be referenced from attributes like `[StringOptions(QueuedEvent.Actions.RegexPattern)]`